py/src/lib.rs

```diff
@@ -357,7 +357,7 @@ impl Scanner {
         Python::with_gil(|py| {
             scan_results_to_py(
                 py,
-                self.inner.scan_file(path).map_err(map_scan_err)?,
+                self.inner.scan_file(path.as_path()).map_err(map_scan_err)?,
```

Comment:
Why is this change necessary? The `scan_file` API is not changing, right?

Reply:
removed in [cca5adf](https://github.com/VirusTotal/yara-x/pull/183/commits/cca5adfaeb3671482a47f5b0a1a8e3f7d2a70fe8)